locales/ru_RU/LC_MESSAGES/duckduckgo.po

```diff
@@ -947,7 +947,7 @@ msgid "Clear source text"
 msgstr "Удалить исходный текст"
 
 msgid "Clear your cookies often?"
-msgstr "Нужно ли часто удалять cookie файлы?"
+msgstr "Часто ли нужно удалять файлы cookie?"
```

Comment:
I checked with an internal employee who speaks Russian because this sounded strange. She suggested this version, which is closer to our intended meaning. FYI, this string is not meant to offer learning material like "How often do you need to clear your cookies", but rather ask the user "Do you clear your cookies often?" and if yes, we offer suggestions to decrease cookie-related messages.

I will go ahead and commit this, but feel free to answer or open a new PR if this seems wrong. Thanks!

```suggestion
msgstr "Часто удаляете cookie файлы?"
```

Reply:
Of course, I saw that something was wrong with the translation of this string. But I decided only to adapt it, since it had already been approved.